src/library/validators.js

```diff
@@ -35,8 +34,12 @@ const checkPassword = str => {
   let force = null
   if (score === Object.keys(info).length) {
     force = 'high'
-  } else if (score >= 3 && (str.length >= 6)) {
-    force = 'medium'
+  } else if (score === 2 && (str.length >= 6)) {
+    force = 'medium-1'
```

Comment:
you can just return the value of force (an integer), and apply the corresponding css class like `force-1` `force-2` etc